async function addAndrenderBook() {
  try {
  const book = await addBook({});
  } catch (error) {
    console.log(error);
  }
}

